pkg/ipamd/ipamd.go

```diff
@@ -322,8 +322,8 @@ func New(k8sapiClient kubernetes.Interface, eniConfig *eniconfig.ENIConfigContro
 		return nil, err
 	}
 	if hypervisorType != "nitro" && c.enableIpv4PrefixDelegation {
-		log.Error("Prefix delegation is not supported non-nitro instance - " + c.awsClient.GetInstanceType())
-		return nil, err
+		log.Warnf("Prefix delegation is not supported on non-nitro instance %s hence falling back to secondary mode", c.awsClient.GetInstanceType())
```

Comment:
nit: secondary mode -> `secondary IP address mode` (or) `default mode`. I'm thinking `default mode` might  be better because I don't think any of our docs talk about secondary mode.

Reply:
Yeah, I am fine with either of the suggestions. Will update the PR.